lib/show.gi

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+##
+#W  show.gi
+##
+##  Contains implementations of standard functions for outputting objects
+##  to streams.
+##
+##  These functions should replace all the View/Print/Display variants, and
+##  in particular get rid of a lot of code duplication, when there are separate
+##  implementations for View and ViewString for instance, and lack of
+##  implementations when View exists, but ViewString doesn't (probably for the
+##  reason above)
+##
+##  output streams will be free to ignore/strip formatting helpers
+##
+##  The naming is still topic of discussion. I renamed Print to Code, because
+##  this makes the requirement to output Code clearer.
+##
+
+############################################################################
+##
+#O  ViewObjStream(<stream>, <obj>)
+##
+InstallMethod(ViewObjStream, "for an output stream, and an object",
```

Comment:
The discussion of the PR calls this `ViewObjToStream`, which I am fine with. But `ViewObjStream`, as used here, is confusing to me.

Another alternative would be to just call this `ViewObj`, too, as there are non two arguments versions of `ViewObj` nor `PrintObj` in the whole GAP library, and all distributed packages.